src/main/resources/static/js/gw.history.js

```diff
@@ -371,7 +371,7 @@ GW.history = {
 
     applyBootstrapTable: function(table_id){
 
-        var table = $("#"+table_id).DataTable({
```

Comment:
why deleting # here?

Reply:
actually the applyBootstrapTable is receiving the table_id but now i made a code change so that it receives the correct table selector. Both the tables in side panel and the main panel are using same table id which leads us to access always the first table when we pass only table_id so i have modified it so now applyBootstrapTable receives the selector instead of table_id. Infact i will change the argument name to table_selector